<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>日程管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon_regis.qrc">
    <normaloff>:/icon.jpg</normaloff>:/icon.jpg</iconset>
  </property>
  <property name="toolTip">
   <string>这是日程管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{background-color:#336666;};</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1">
       <item row="0" column="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>用户名: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(102, 153, 153);</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/蛙蛙.jpg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="icon2.qrc">:/蛙蛙.jpg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_user">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>AniMe Matrix - MB_EN</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>314</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>AniMe Matrix - MB_EN</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>日程管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <widget class="QPushButton" name="btn_add">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>130</width>
            <height>40</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>AniMe Matrix - MB_EN</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(102, 153, 153);</string>
          </property>
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_del">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>130</width>
            <height>40</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>AniMe Matrix - MB_EN</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(102, 153, 153);</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_mod">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>130</width>
            <height>40</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>AniMe Matrix - MB_EN</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(102, 153, 153);</string>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>46</x>
            <y>330</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(102, 153, 153);</string>
          </property>
          <property name="text">
           <string>注销</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_4" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>801</width>
            <height>401</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>id</string>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>任务名称</string>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>开始时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>提醒时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>优先级</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>分类</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_6" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <widget class="QPushButton" name="btn_check">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>8</y>
         <width>75</width>
         <height>25</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>10</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>所有</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>低</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="line_taskname">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>11</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>日程名称</string>
       </property>
       <property name="placeholderText">
        <string>任务名称</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_classification">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>10</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>分类</string>
       </property>
       <property name="placeholderText">
        <string>分类</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>9</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>AniMe Matrix - MB_EN</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>查询时间</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_7" native="true">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>0</y>
         <width>210</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <widget class="QDateTimeEdit" name="time1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_8" native="true">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>0</y>
         <width>211</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <widget class="QDateTimeEdit" name="time2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>194</width>
          <height>22</height>
         </rect>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon2.qrc"/>
  <include location="icon_regis.qrc"/>
 </resources>
 <connections/>
</ui>
